include/envoy/config/grpc_mux.h

```diff
@@ -99,7 +101,8 @@ class GrpcMux {
   virtual GrpcMuxWatchPtr addWatch(const std::string& type_url,
                                    const std::set<std::string>& resources,
                                    SubscriptionCallbacks& callbacks,
-                                   OpaqueResourceDecoder& resource_decoder) PURE;
+                                   OpaqueResourceDecoder& resource_decoder,
+                                   const bool use_prefix_matching) PURE;
```

Comment:
@chaoqin-li1123 some variant of this might be useful for glob matching when subscribing to glob collections in the work you are doing. For `udpa://` URLs, it's not a prefix on the path component (ignoring context parameters, which should be exact match).